keyboards/osprette/info.json

```diff
@@ -0,0 +1,70 @@
+{
+    "keyboard_name": "The Osprette",
+    "manufacturer": "S'mores",
+    "url": "https://github.com/smores56/osprette",
+    "maintainer": "@smores56",
+    "usb": {
+      "vid": "0xBEEF",
+      "pid": "0x5052",
+      "device_version": "1.0.0"
+    },
+    "development_board": "elite_c",
```

Comment:
Must an Elite-C development board be used for this keyboard, or would a Pro Micro development board suffice? 🤔

Reason I ask: I'm not seeing any other settings, asides from `"development_board"`, that are specific to Elite-C.

Reply:
It's a pro-micro compatible board, that's it. I'm okay to remove it if you think that's better!

Reply:
I'll do one better, and put code suggestion in to change value of `"development_board"` 😉